1.shift/Pocket_beagle/shift.c

```diff
@@ -12,15 +11,15 @@
  /****************************************************************
  * global
  ****************************************************************/
-	unsigned int data,clk,latch;
-	unsigned int  registers[8];
-    int data1,clk1,latch1;
+unsigned int data,clk,latch;
+unsigned int  registers[8];
+int data1,clk1,latch1;
 
  /****************************************************************
  * Constants
  ****************************************************************/
 
-#define SYSFS_GPIO_DIR "/sys/class/gpio"
+#define SYSFS_GPIO_DIR "/home/kshitiz/git_gsoc/1.shift/Pocket_beagle"
```

Comment:
First of all Thanks for contributing in my project. This file is in  the root directory and need not be change